eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -487,11 +487,10 @@ asset_name = bytes .size (0..32)
 negInt64 = -9223372036854775808 .. -1
 posInt64 = 1 .. 9223372036854775807
 nonZeroInt64 = negInt64 / posInt64 ; this is the same as the current int64 definition but without zero
-natNum = 1 .. 4294967295 ; non-zero 32bit uint
 
 positive_coin = 1 .. 18446744073709551615
 
-value = positive_coin / [positive_coin,multiasset<natNum>]
+value = positive_coin / [positive_coin,multiasset<positive_coin>]
```

Comment:
@JaredCorduan , I think the correct change here would be this:

```suggestion
value = positive_coin / [positive_coin,multiasset<nonZeroInt64>]
```

Because in the code, technically, the multi-asset decoding in the value is completely equal to that of the mint. See here: `decodeValuePair` uses `decodeMultiAssetMaps` internally: https://github.com/input-output-hk/cardano-ledger/blob/c1603995da7c57d95371844191d235b05bb476b7/eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs#L325 and `decodeMultiAssetMint` uses same exact `decodeMultiAssetMaps` internally as well: https://github.com/input-output-hk/cardano-ledger/blob/c1603995da7c57d95371844191d235b05bb476b7/eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs#L349

The mint is defined as 
```
mint = multiasset<nonZeroInt64>
```

So the multi-asset in the value should just be the same

Reply:
The `positive_coin`, used in your fix has maximum value of 18446744073709551615, which is not equal to the maximum value of the `posInt64` - 9223372036854775807, which I think is the limit to how much of an asset can be sent